config.go

```diff
@@ -531,8 +495,12 @@ func loadConfig() (*config, error) {
 			litecoinChain.String())
 
 		// Finally we'll register the litecoin chain as our current
-		// primary chain.
+		// primary chain. We'll temporarily increase the maximum channel
+		// and payment size allowed since the current limits are not
+		// very useful due to the price difference.
 		registeredChains.RegisterPrimaryChain(litecoinChain)
+		maxFundingAmount *= 60
+		maxPaymentMSat *= 60
```

Comment:
Instead of doing this scaling here, we can set local `maxFundingAmount` or `maxPaymentMSat` vars based on which chain is active

Reply:
Fixed.